agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/configuration/ConnectionStringBuilder.java

```diff
@@ -34,6 +34,8 @@ class ConnectionStringBuilder {
   // visible for testing
   static final int CONNECTION_STRING_MAX_LENGTH = 4096;
 
+  private String parsedFrom;
```

Comment:
can we rename this to be consistent with ConnectionString?

Reply:
done